### DISCUSSION TITLE
Stylish API

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#22913] [Quarkus Blog/Website]
# Stylish API

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/stylish-api/

**Comment by spartanhooah** [comment]
This doesn't seem to work anymore.

**Reply by spartanhooah** [reply]
By which I mean the theming.

**Reply by phillip-kruger** [reply]
That is not good :( I'll have look. Do you have any errors ? Either in the server log or the browser console ?

**Reply by spartanhooah** [reply]
No errors or logs. It just doesn't change the UI at all.

**Reply by phillip-kruger** [reply]
OK I'll have a look a.s.a.p

**Reply by spartanhooah** [reply]
Apparently this is only an issue when using a yml file for application properties.

**Reply by spartanhooah** [reply]
This might be a bigger issue: https://github.com/quarkusio/quarkus/discussions/49111

**Reply by phillip-kruger** [reply]
If I understand correctly this is now solved ? @spartanhooah please confirm

**Reply by spartanhooah** [reply]
Yes, resolved.
